blocks/feed-newsroom/feed-newsroom.css

```diff
@@ -0,0 +1,89 @@
+/* eslint no-empty: "info" */
+
+main .section.feed-newsroom-container > .section-container {
+    max-width: 1200px;
```

Comment:
Should `narrow` section variant help here? Typically blocks should not influence their parent styles (like the feed-newsroom block influencing it's parent's width)